### PR TITLE
Add 'updated_since' param for TicketAPI.list_ticket() method

### PR DESCRIPTION
**Pull request by jatzz10**

### CODE REVIEW

File: freshdesk/v2/api.py [hunk 1]
```diff
@@ -135,8 +135,12 @@ def list_tickets(self, **kwargs):
             url += "?filter=%s&" % filter_name
         else:
             url += "?"
-        page = kwargs.get("page", 1)
-        per_page = kwargs.get("per_page", 100)
+
+        if "updated_since" in kwargs:
```
Comment (sjkingo) [inline freshdesk/v2/api.py:139]: Could you please add a test and small line in the README under `list_ticket()` to show this is now supported? From looking at this I don't know what format it should be in.

File: freshdesk/v2/api.py [hunk 2]
```diff
@@ -135,8 +135,12 @@ def list_tickets(self, **kwargs):
...
+            url += "updated_since=%s&" % kwargs["updated_since"]
+
+        page = 1 if not "page" in kwargs else kwargs["page"]
```
Comment (sjkingo) [inline freshdesk/v2/api.py:142]: You've brought back in some old code here, we changed this in [83b97d2](https://github.com/sjkingo/python-freshdesk/commit/83b97d27d021f1d994a7c2b46eeed1573bb8a605#diff-33bd36adc6a6677fdf3d3d6fe640eba9R138).

File: freshdesk/v2/api.py (hunk 1 above)
Comment (jatzz10) [reply freshdesk/v2/api.py:139]: I have updated the PR just now. Can you please check if everything looks good?
Thanks!

File: freshdesk/v2/api.py (hunk 2 above)
Comment (jatzz10) [reply freshdesk/v2/api.py]: This has been fixed now.

**Comment by jatzz10** [comment]
Hey @sjkingo , is there any other change required from my side? Please let me know.

**Comment by sjkingo** [comment]
I'll have a good look today and merge in. Thanks!

**Comment by sjkingo** [comment]
Looks great! I've pushed this as 1.3.3 on pypi. Thanks!

**Comment by jatzz10** [comment]
Awesome. Thanks! :)

**Pull request merged**